base_external_dbsource/models/base_external_dbsource.py

```diff
@@ -322,7 +322,7 @@ def _get_adapter_method(self, method_prefix):
         """
 
         self.ensure_one()
-        method = "{}_{}".format(method_prefix, self.connector)
+        method = f"{method_prefix}_{self.connector}"
```

Comment:
This change must be done on migration commit, not in this commit.

Reply:
Not sure, but this change was added by `pre-commit` during pre-commit auto fixes stuff.